src/EncompassRest/Webhook/Webhook.cs

```diff
@@ -246,18 +246,15 @@ public Task UpdateSubscriptionRawAsync(string subscriptionId, string subscriptio
             }
         }
 
-        public Task DeleteSubscriptionAsync(string subscriptionId) => DeleteSubscriptionAsync(subscriptionId, CancellationToken.None);
+        public Task<bool> DeleteSubscriptionAsync(string subscriptionId) => DeleteSubscriptionAsync(subscriptionId, CancellationToken.None);
 
-        public async Task DeleteSubscriptionAsync(string subscriptionId, CancellationToken cancellationToken)
+        public async Task<bool> DeleteSubscriptionAsync(string subscriptionId, CancellationToken cancellationToken)
         {
             Preconditions.NotNullOrEmpty(subscriptionId, nameof(subscriptionId));
 
             using (var response = await Client.HttpClient.DeleteAsync($"{s_apiPath}/subscriptions/{subscriptionId}", cancellationToken).ConfigureAwait(false))
             {
-                if (!response.IsSuccessStatusCode)
-                {
-                    throw await RestException.CreateAsync(nameof(DeleteSubscriptionAsync), response).ConfigureAwait(false);
-                }
+                return response.IsSuccessStatusCode;
```

Comment:
That's how we do it for all the other delete api's. Essentially I'd like it to return a bool because it should be a safe operation, i.e. no exceptions thrown, when calling delete multiple times on a resource.